vpython/no_notebook.py

```diff
@@ -231,10 +233,32 @@ def onClose(self, wasClean, code, reason):
     else:
         __server = HTTPServer(('', __HTTP_PORT), serveHTTP)
         # or webbrowser.open_new_tab()
-        _webbrowser.open('http://localhost:{}'.format(__HTTP_PORT))
+        if _browsertype == 'default':  # uses default browser
+            _webbrowser.open('http://localhost:{}'.format(__HTTP_PORT))
+
 except:
     pass
 
+
+if platform.python_implementation() == 'PyPy' and _browsertype == 'pyqt':
```

Comment:
@atitus -- does this look OK to you for PyPy error handling? It doesn't avoid an error but it does give a fairly explicit error message. I'm a little reluctant to have PyPy automagically ignore the pyqt browser setting because I imagine it will lead to confusion about why the regular browser is opening, but it would be straightforward to make that happen.

Reply:
Yes, that's fine with me. I don't know for sure that using PyPy and the pyqt browser are incompatible. I just know that the way I originally implemented pyqt in no_notebook.py was solely for non-PyPy cases.  It's possible that you or someone else might implement pyqt in a way that is compatible with PyPy.

Reply:
OK; I'll merge tomorrow morning to give folks a bit of time to make comments.